frontend/Dockerfile.amd64

```diff
@@ -2,6 +2,7 @@ FROM golang:buster AS step_0
 ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on
 WORKDIR /root
 COPY . .
+RUN go generate
```

Comment:
On my system `go generate` doesn't work until I download the `go-bindata` binary. Perhaps do something like `go get -u github.com/go-bindata/go-bindata/...`?

Reply:
I'm rubbish at docker :)

I've updated the Dockerfile to reference the right go-bindata version. I also took the opportunity to parameterise the Dockerfile, seemed a bit pointless duplicating changes across multiple files. 

Note that most of the GOOS=linux/GOARCH=xxx combinations that are in the .travis file don't actually work because they aren't supported by go (and the proxy build is hardcoded for amd64 anyway) but I left all that in there.